### ISSUE TITLE
SyntaxError: Unexpected token function

### ISSUE THREAD

**Issue by Darkside73** [nuxt-community/sentry-module#3]
I'm getting error on ```nuxt``` starting
```
node_modules/@nuxtjs/sentry/lib/sentry.js:4
module.exports = async function sentry (options) {
                       ^^^^^^^^

SyntaxError: Unexpected token function
```
```nuxt 1.0.0-rc11```

**Comment by pi0** [comment]
Hi. I think this is because you are using node 6. Please simply upgrade to 8 for fix :)

**Comment by Darkside73** [comment]
Yes, this error has gone. But I don't see any error reports in sentry

**Comment by DiederikvandenB** [comment]
@Darkside73 without more code it is hard to debug your problem. In case of your application throwing a serverside error, try this:

Start by checking if this module's errorHandler is even used. You can do this by adding a simple `console.error('something');` [here](https://github.com/nuxt-community/sentry-module/blob/master/lib/sentry.js#L19).

Rebuild NuxtJS and reload the page that is throwing the error. Check your terminal for the occurrence of the `something` string. If this appears, it means that the `Raven.captureException(err, { req })` method is called and therefore the error should be logged. If it doesn't, please check your Sentry settings.

Bottom line: provide me with more details and I am sure I'll be able to help you out ;).

**Comment by Darkside73** [comment]
Updated to master branch version of module and it works now. I think it should be in the install docs. Thanks for pointing me out anyway)

**Comment by DiederikvandenB** [comment]
I have published a new version to NPM. Everything should work fine.
